packages/client/package.json

```diff
@@ -46,7 +46,7 @@
         "test-types": "npm run test-browser-types",
         "test-browser-types": "tsc -b tsconfig.browser.json",
         "coverage": "jest --coverage",
-        "test-integration": "jest --verbose --useStderr --maxWorkers=2 --forceExit test/integration",
+        "test-integration": "jest --verbose --useStderr --maxWorkers=8 --forceExit test/integration",
```

Comment:
If there is a specific `maxWorkers` setting that works well in CI, I would look into configuring it into Github Actions YAML instead of here. Reason being is that it is very environment specific what value works well. (For example, if you don't have many cores on your machine, setting the value high will actually slow down the test suite.)

You can pass `maxWorkers` to the command as follows, for example:
```
npm run test-integration -- --maxWorkers=8
``´

Reply:
ah yes, will remove this, it was just a test to see if ci would run them in parallel